.github/workflows/ci.yml

```diff
@@ -1199,6 +1199,96 @@ jobs:
         timeout-minutes: 5
         run: cat /tmp/flv_sc.log
 
+  mirroring_smoke_test_e2e:
```

Comment:
This doesn' need to have separate cluster for it.  Can just add to existing BATS test

Reply:
this is from: https://github.com/infinyon/fluvio/pull/4028

I separated e2e with home and remote tests and simple cli tests there

Reply:
ok.  will revisit once that is merged